lang/de.json

```diff
@@ -163,7 +163,7 @@
 		"uploadPermissions": "Datei hochladen nicht erlaubt."
 	},
 	"chooseResizedImage": {
-		"title": "[MISSING]Choose Resized",
+		"title": "Choose Resized",
```

Comment:
Would you have an idea for a more German equivalent here? ;) This is the label of the toolbar button that lets you choose an image, resize/scale it and send the resized/scaled version to another application like CKEditor. I know it's tough when it also needs to be short :/